.github/workflows/golangci-lint.yml

```diff
@@ -20,11 +20,11 @@ jobs:
       - name: install Go
         uses: actions/setup-go@v2
         with:
-          go-version: 1.18.x
+          go-version: 1.20.x
       - name: Install snmp_exporter/generator dependencies
         run: sudo apt-get update && sudo apt-get -y install libsnmp-dev
         if: github.repository == 'prometheus/snmp_exporter'
       - name: Lint
-        uses: golangci/golangci-lint-action@v3.2.0
```

Comment:
What is the reason that we are explicitly calling out the full version here? We could use `v3` and each run should reference the latest. I assume we had some issue in the past?

Reply:
Good question, I've always seen these use full semver. I just recently tried to use a major version like that in another action, but it failed to work without full semver.